src/components/JournalIndexPage/index.jsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Button } from '@edx/paragon';
+import JournalAboutPage from '../JournalAboutPage';
+import HeroBanner from '../../components/HeroBanner';
+
+import './JournalIndexPage.scss';
+
+
+const JournalCard = ({
+  journalAboutId,
+  title,
+  cardImageUrl,
+  organization,
+  authorized,
+}) => (
+  <Link className="journal-card-link" to={{ pathname: authorized ? `/${journalAboutId}` : `/${journalAboutId}/about` }}>
+    <div className="journal-card" >
```

Comment:
So this may be a little late... but Bootstrap has a card component (not react component) that might make this easier/cleaner. https://getbootstrap.com/docs/4.1/components/card/ . I'll leave it up to you though, I'm fine either way.